src/rules/iddataweb-verification-workflow.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * @title ID DataWeb Verification Workflow
+ * @overview Verify your user's identity in 180+ countries with ID DataWeb's adaptive Verification Workflows.
+ * @gallery true
+ * @category marketplace
+ *
+ * This configuration allows you to add an ID DataWeb verification workflow to the user’s first time login experience. This will allow your organization to ensure the user is really who they claim to be, aligning with KYC/AML requirements, and mitigating fraud attempts.
+ *
+ */
+
+function iddatawebVerificationWorkflow(user, context, callback) {
+  const { Auth0RedirectRuleUtilities } = require("@auth0/rule-utilities@0.1.0");
+
+  const ruleUtils = new Auth0RedirectRuleUtilities(
+    user,
+    context,
+    configuration
+  );
+
+  const {
+    IDDATAWEB_BASE_URL,
+    IDDATAWEB_CLIENT_ID,
+    IDDATAWEB_CLIENT_SECRET,
+    IDDATAWEB_LOG_JWT,
+    IDDATAWEB_ALWAYS_VERIFY,
+    IDDATAWEB_PREFILL_ATTRIBUTES,
+  } = configuration;
+
+  const idwTokenEndpoint = `${IDDATAWEB_BASE_URL}/axn/oauth2/token`;
+  const idwAuthorizeEndpoint = `${IDDATAWEB_BASE_URL}/axn/oauth2/authorize`;
+  const auth0ContinueUrl = `https://${context.request.hostname}/continue`;
+
+  // initialize app metadata
+  user.app_metadata = user.app_metadata || {};
+  user.app_metadata.iddataweb = user.app_metadata.iddataweb || {};
+
+  // if the user is already verified and we don't need to check, exit
+  if (
+    user.app_metadata.iddataweb.verificationResult === "verified" &&
+    IDDATAWEB_ALWAYS_VERIFY === "off"
```

Comment:
The other two options look for `"on"` as the value. Should we switch this to the same? The unset condition should be the better default.

Reply:
understood, i will rework this.